lucet-module/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "lucet-module-data"
+name = "lucet-module"
 version = "0.1.1"
-description = "A structured interface for Lucet module data and metadata"
```

Comment:
This and `lucet-module-data/src/lib.rs` are the only non-rename changes. Figured updating these to match the name is appropriate